.github/workflows/auto_workflow_approve.yml

```diff
@@ -0,0 +1,15 @@
+name: Automatic Approve Workflows
+on:
+  schedule: 
+    - cron: "*/10 * * * *"
+jobs:
+  automatic-approve:
+    name: Automatic Approve Workflows
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Automatic Approve
+        uses: mheap/automatic-approve-action@c18642bf35643065692b1108cc3becfa4804087b
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          workflows: "ci_suite.yml,gbp.yml"
+          dangerous_files: "build.bat"
```

Comment:
A quick glance at mheap/automatic-approve-action's code suggests that this is case-sensitive. The file is actually called `"BUILD.bat"`. Not that it really matters, because there's a million other script files that can be modified to get around this.

The change from GitHub was (at least partially) intended to protect repositories from running malicious code that they'll then be held accountable for... should we really add a workaround like this? It might seem unlikely that it'll be abused, but I imagine the list of possible repositories for miners to abuse becomes a lot shorter when they just have to search for these actions being present.

Reply:
> but I imagine the list of possible repositories for miners to abuse becomes a lot shorter when they just have to search for these actions being present

this was my thoughts